### PR TITLE
Es output

### PR DESCRIPTION
**Pull request by stefvhuynh**

### CODE REVIEW

**Comment by stefvhuynh** [comment]
@ryan-roemer, @tptee,  es build :)

File: package.json [hunk 1]
```diff
@@ -3,25 +3,30 @@
...
     "preversion": "npm run check",
-    "version": "npm run clean && npm run build && git add -A dist",
+    "version": "npm run clean && npm run build",
     "demo": "webpack-dev-server --port 3000 --config config/webpack/demo/webpack.config.demo.dev.js --colors",
     "build-demo": "webpack --bail --config config/webpack/demo/webpack.config.demo.js",
     "start-ssr-demo": "nodemon --watch demo demo/server/index.js",
     "clean-dist": "rimraf dist",
-    "build-dist-min": "webpack --bail --config config/webpack/webpack.config.js",
-    "build-dist-dev": "webpack --bail --config config/webpack/webpack.config.dev.js",
+    "build-dist-min": "BABEL_ENV=commonjs webpack --bail --config config/webpack/webpack.config.js",
```
Comment (ryan-roemer) [inline package.json]: Webpack should use ES version _not_ `commonjs`.

**Comment by stefvhuynh** [comment]
@ryan-roemer, addressed comments

**Comment by stefvhuynh** [comment]
@ryan-roemer, could i get another quick review and a lgtm on this? thanks :)

**Comment by ryan-roemer** [comment]
@tptee -- What is this bitHound thing failing on? It says `bitHound - Dependencies — 11 failing dependencies.` as a red X, but the link says: https://www.bithound.io/github/FormidableLabs/redux-little-router/a999abc839d077e53f2475110f87dfae4ab5c453/dependencies/npm#filter-failing-dep

Can we either fix this or turn it off? (My motivation here is that having regularly failing CI things is super bad because folks start to ignore CI generally, even the important parts that work 😉  )

File: package.json [hunk 2]
```diff
@@ -3,25 +3,30 @@
...
+  "jsnext:main": "es/index.js",
   "scripts": {
     "preversion": "npm run check",
-    "version": "npm run clean && npm run build && git add -A dist",
-    "demo": "webpack-dev-server --port 3000 --config config/webpack/demo/webpack.config.demo.dev.js --colors",
-    "build-demo": "webpack --bail --config config/webpack/demo/webpack.config.demo.js",
-    "start-ssr-demo": "nodemon --watch demo demo/server/index.js",
+    "version": "npm run clean && npm run build",
+    "demo": "BABEL_ENV=commonjs webpack-dev-server --port 3000 --config config/webpack/demo/webpack.config.demo.dev.js --colors",
+    "build-demo": "BABEL_ENV=commonjs webpack --bail --config config/webpack/demo/webpack.config.demo.js",
```
Comment (ryan-roemer) [inline package.json]: I don't think we need `BABEL_ENV=commonjs` for anything pure webpack -- so not for `demo` or `build-demo`

Comment (stefvhuynh) [reply package.json]: The demo actually didn't work unless it was with the `commonjs` env. When I went with the `es` build for demo (which is the default now), it complained about `export` not being a recognized token.

Comment (ryan-roemer) [reply package.json]: Hmmmm... Let me take a look at this and figure out what's going wrong...

**Comment by tptee** [comment]
Removed bithound–nothing but a pain. Have been trying for about a year now to get rid of codacy...

**Pull request merged**